hw3/01-characters.js

```diff
@@ -1,4 +1,26 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+const c_list = document.getElementById('charList');
+        fetch(url)
+            .then(response => response.json())
+            .then(characters => {
+                characters.forEach(character => {
+                    const singleChar = document.createElement('div');
+                    singleChar.className = 'single_char';
+                    const char_pic = document.createElement('img');
+                    char_pic.className = 'charPic';
+                    char_pic.src = character.imageUrl;
+                    const char_stats = document.createElement('div');
+                    const char_name = document.createElement('p');
+                    char_name.textContent = character.fullName;
+                    const char_title = document.createElement('p');
+                    char_title.textContent = character.title;
+                    char_stats.appendChild(char_name);
+                    char_stats.appendChild(char_title);
+                    singleChar.appendChild(char_pic);
+                    singleChar.appendChild(char_stats);
```

Comment:
Using the regular append() method multiple children elements can be appended to parent at the same time. This is not necessary, but it would save a few lines while still being readable.

The append() method works like appendChild(), but lets you pass in multiple items.

```suggestion
                    char_stats.append(char_name, char_title);
                    singleChar.append(char_pic, char_stats);
```